src/register/RegistrationPage.jsx

```diff
@@ -282,104 +383,146 @@ const RegistrationPage = (props) => {
+          ? (
+            <div className="mw-xs mt-5 text-center">
+              <Spinner animation="border" variant="primary" id="tpa-spinner" />
+            </div>
+          ) : (
+            <div
+              className={classNames(
+                'mw-xs mt-3',
+                { 'w-100 m-auto pt-4 main-content': registrationEmbedded },
               )}
-              <ConfigurableRegistrationForm
-                email={formFields.email}
-                fieldErrors={errors}
-                formFields={configurableFormFields}
-                setFieldErrors={registrationEmbedded ? setTemporaryErrors : setErrors}
-                setFormFields={setConfigurableFormFields}
-                autoSubmitRegisterForm={autoSubmitRegForm}
-                fieldDescriptions={fieldDescriptions}
+            >
+              <ThirdPartyAuthAlert
+                currentProvider={currentProvider}
+                platformName={platformName}
+                referrer={REGISTER_PAGE}
               />
-              <StatefulButton
-                id="register-user"
-                name="register-user"
-                type="submit"
-                variant="brand"
-                className="register-button mt-4 mb-4"
-                state={submitState}
-                labels={{
-                  default: buttonLabel,
-                  pending: '',
-                }}
-                onClick={handleSubmit}
-                onMouseDown={(e) => e.preventDefault()}
+              <RegistrationFailure
+                errorCode={errorCode.type}
+                failureCount={errorCode.count}
+                context={{ provider: currentProvider, errorMessage: thirdPartyAuthErrorMessage }}
+                multiStepRegistrationPageStep={multiStepRegistrationPageStep}
               />
-              {!registrationEmbedded && (
-                <ThirdPartyAuth
-                  currentProvider={currentProvider}
-                  providers={providers}
-                  secondaryProviders={secondaryProviders}
-                  handleInstitutionLogin={handleInstitutionLogin}
-                  thirdPartyAuthApiStatus={thirdPartyAuthApiStatus}
+              <Form id="registration-form" name="registration-form">
+                {(multiStepRegistrationExpVariation === MULTI_STEP_REGISTRATION_EXP_VARIATION
+                  && multiStepRegistrationPageStep === SECOND_STEP) && (
+                  <p className="h3 mb-4">
+                    {formatMessage(messages['multistep.registration.username.second.step.guideline.content'])}
+                  </p>
+                )}
+                {((multiStepRegistrationExpVariation === MULTI_STEP_REGISTRATION_EXP_VARIATION
+                    && multiStepRegistrationPageStep === THIRD_STEP)
+                    || (multiStepRegistrationExpVariation === CONTROL && multiStepRegistrationPageStep === SECOND_STEP))
```

Comment:
Why do we need this condition for control variation to show username guideline message copy?
`(multiStepRegistrationExpVariation === CONTROL && multiStepRegistrationPageStep === SECOND_STEP) `